Gems/Atom/Feature/Common/Assets/Shaders/PostProcessing/LookModificationTransform.azsl

```diff
@@ -42,6 +43,105 @@ option bool o_enableExposureControlFeature = false;
 // Option shader variable to enable color grading LUT.
 option bool o_enableColorGradingLut = false;
 
+// Controls the sampling quality of the blended LUT. Setting this higher can improve the quality of particularly tricky luts.
+// 0 - linear
+// 1 - 7 tap b-spline
+// 2 - 19 tap b-spline
+[[range(0, 2)]]
+option uint o_lutSampleQuality = 0;
+
+// Sample a 3dtexture with a 7 or 19 tap B-Spline. Consider ripping this out and putting in a more general location.
+// This function samples a 4x4x4 neighborhood around the uv. Normally this would take 64 samples, but by taking
+// advantage of bilinear filtering this can be done with 27 taps on the edges between pixels. The cost is further
+// reduced by dropping either the 8 corners (19 total taps) or also dropping the 12 edges (7 total taps).
+float4 SampleBSpline3D(Texture3D<float4> texture, SamplerState linearSampler, float3 uv, float3 textureSize, float3 rcpTextureSize)
```

Comment:
Nit: no need to do this for this change, but I think refactoring this later to express weighted taps as matrices might make it easier to read (7/19 tap filter is no joke lol)

Reply:
Yeah I definitely want to spend more time on this in the future. I'm not satisfied with the quality of the 7 tap and I'm also not satisfied with the cost of the 19 tap. Being able to switch out kernels easily would be nice. I'll be creating some issues to follow up on later.